src/cpp/rtps/participant/RTPSParticipantImpl.cpp

```diff
@@ -1658,6 +1666,17 @@ bool RTPSParticipantImpl::createSendResources(
     return true;
 }
 
+void RTPSParticipantImpl::setupExternalLocators(
+        Endpoint* pend)
```

Comment:
Use a significant name for the argument
```suggestion
        Endpoint* endpoint)
```

Reply:
Same as above, but also :+1: